provisioning/stream/kubernetes/templates/storage-writer-reader/statefulset.yaml

```diff
@@ -147,7 +152,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
             - name: STREAM_HOSTNAME
-              value: "$(STREAM_NODE_ID).stream.pod.cluster.local"
+              value: "localhost"
             - name: STREAM_ETCD_PASSWORD
               valueFrom:
```

Comment:
Hostname is required configuration field, I set it to `localhost` for pods without service.